public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts

```diff
@@ -377,7 +383,7 @@ export function createSceneVariableFromVariableModel(variable: TypedVariableMode
   } else if (variable.type === 'textbox') {
     return new TextBoxVariable({
       ...commonProperties,
-      value: variable.query,
+      value: convertToString(variable.current?.value ?? ''),
```

Comment:
textbox can't have a value other than string. do have an instance of a textbox with array?

Reply:
It should be always a string but the type of value is `string | string[]` I will change it to 
```suggestion
     value: variable.current?.value[0] ?? '',
```